src/browser/modules/Editor/EditorFrame.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2002-2020 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useState, useEffect } from 'react'
+import { withBus } from 'react-suber'
+import { Bus } from 'suber'
+import Editor from './Editor'
+import { Frame, FrameHeader, FrameHeaderText, UIControls } from './styled'
+import {
+  EDIT_CONTENT,
+  SET_CONTENT,
+  GET_CONTENT,
+  EXPAND
+} from 'shared/modules/editor/editorDuck'
+import { FrameButton } from 'browser-components/buttons'
+import {
+  ExpandIcon,
+  ContractIcon,
+  CloseIcon,
+  UpIcon,
+  DownIcon
+} from 'browser-components/icons/Icons'
+
+type EditorSize = 'CARD' | 'LINE' | 'FULLSCREEN'
+type EditorFrameProps = { bus: Bus }
+
+export function EditorFrame({ bus }: EditorFrameProps) {
```

Comment:
Been going back and forth if I should reuse more of the existing frame code. I think that would have been better, we'll see when the frames get moved to look like the editor how much code sharing feels natural